src/objects.js

```diff
@@ -1,41 +1,55 @@
 const createPerson = (name, age) => {
-  // your code here
+  return { name, age }
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  return (person.age > 65);
 };
 
 const getAges = people => {
-  // your code here
+  return people.map((person) => {
+    return person.age;
+  });
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(individual => {
+    return individual.name === name;
+  });
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter((individual) => {
+    return individual.manufacturer === 'Honda';
+  });
 };
 
 const averageAge = people => {
-  // your code here
+  return (people.reduce((a,b) => {
+    return a + b.age;
+  }, 0) / people.length);
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    "name": name,
+    "age": age,
+    "introduce": (nameTwo) => {
+      return `Hi ${nameTwo}, my name is ${name} and I am ${age}!`
```

Comment:
Can be made a bit cleaner using object property value shorthand, a new feature of ES6 (Comparison between ES5 and 6 [here](https://alligator.io/js/object-property-shorthand-es6/)):
```suggestion
    name,
    age,
    "introduce": nameTwo => {
      return `Hi ${nameTwo}, my name is ${name} and I am ${age}!`
```